app/vue/src/client/preview/render.js

```diff
@@ -28,9 +28,9 @@ export default function render({
 }) {
   Vue.config.errorHandler = showException;
 
-  const component = story();
+  const element = storyFn();
```

Comment:
@ndelangen for the sake of clarity: `component` is more correct, as the return value of the story function should be a vue component. the term `element` in vue is used to refer to DOM elements